packages/modules/payment/src/migrations/Migration20250924135437.ts

```diff
@@ -0,0 +1,33 @@
+import { Migration } from "@mikro-orm/migrations"
+import { ulid } from "ulid"
+
+export class Migration20250924135437 extends Migration {
+  override async up(): Promise<void> {
+    const existingReasonIds = await this.execute(`
+      SELECT id FROM "refund_reason"
```

Comment:
nit(non-blocking): don't think we need to fetch all
```suggestion
      SELECT 1 FROM "refund_reason"
```